apps/web/app/(org)/dashboard/caps/Caps.tsx

```
@@ -190,85 +195,78 @@ export const Caps = ({
 
-  const deleteCap = async (capId: string) => {
-    try {
-      await deleteVideo(capId);
+  const deleteCap = useEffectMutation({
+    mutationFn: (id: Video.VideoId) => withRpc((r) => r.VideoDelete(id)),
+    onSuccess: Effect.fn(function* () {
       toast.success("Cap deleted successfully");
-      refresh();
-    } catch (error) {
+      router.refresh();
+    }),
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)